src/containers/Analytics.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <div class="body-container" :class="{ 'no-data': !hasData() }">
```

Comment:
if there's no data can you say "rip no data or there was an error" something of the sorts

Reply:
The reason why I don't do this is to avoid showing that message to users that are not on CoursePlan, who we might want to avoid even knowing this sort of data exists. Let me know if there is an appropriate message that should show here though!